components/feature/sitepermissions/src/main/res/values-kn/strings.xml

```diff
@@ -1,11 +1,32 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <!-- Title of a dialog for a notification permission request. -->
+    <string name="mozac_feature_sitepermissions_notification_title">ಅಧಿಸೂಚನೆಗಳನ್ನು ಕಳುಹಿಸಲು% 1 $ s ಅನ್ನು ಅನುಮತಿಸುವುದೇ?</string>
```

Comment:
@Delphine it looks like we have an issue with the parameter. It should be `%1$s`

Reply:
Fixed in Pontoon, should be available after next Pontoon sync